app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -226,9 +233,8 @@ const ClaimScreen: React.FC = () => {
       <View style={styles.buttonWrapper}>
         <Button
           title={isVerifying ? "Save & Verify" : "Save"}
-          disabled={!canSave}
+          disabled={!canSave || disableButton}
```

Comment:
Why not set disabled to just disableButton or canSave, why both?

Reply:
I think it still needs to meet the requirements of canSave for the button not to be disabled :) unlesss I put my logic in canSave?

Reply:
See new comment